json/v3/schema.json

```diff
@@ -1,637 +1,245 @@
-                "required": [
-                  "ID", 
-                  "IDScheme"
-                ]
-              }
-            }, 
-            "required": [
-              "name"
-            ]
+            "$ref": "#/definitions/IdentifierType"
           }
         }
-      }, 
+      },
       "required": [
-        "Identifier", 
-        "Type"
+        "Name"
       ]
-    }, 
-    "Target": {
-      "$id": "/properties/Target", 
-      "type": "object", 
+    },
+    "ObjectType": {
+      "type": "object",
+      "additionalProperties": false,
       "properties": {
         "Identifier": {
-          "$id": "/properties/Target/properties/Identifier", 
-          "type": "object", 
-          "properties": {
-            "ID": {
-              "$id": "/properties/Target/properties/Identifier/properties/ID", 
-              "type": "string", 
-              "title": "The Id Schema.", 
-              "description": "An explanation about the purpose of this instance.", 
-              "default": "", 
-              "examples": [
-                "10.1016/j.ijmedinf.2009.08.006"
-              ]
-            }, 
-            "IDScheme": {
-              "$id": "/properties/Target/properties/Identifier/properties/IDScheme", 
-              "type": "string", 
-              "title": "The Idscheme Schema.", 
-              "description": "An explanation about the purpose of this instance.", 
-              "default": "", 
-              "examples": [
-                "doi"
-              ]
-            }, 
-            "IDURL": {
-              "$id": "/properties/Target/properties/Identifier/properties/IDURL", 
-              "type": "string", 
-              "title": "The Idurl Schema.", 
-              "description": "An explanation about the purpose of this instance.", 
-              "default": "", 
-              "examples": [
-                "https://www.ncbi.nlm.nih.gov/pubmed/19783203"
-              ]
-            }
-          }, 
-          "required": [
-            "ID", 
-            "IDScheme"
-          ]
-        }, 
+          "$ref": "#/definitions/IdentifierType",
+          "description": "Identifier of the source or target object."
+        },
         "Type": {
-          "$id": "/properties/Target/properties/Type", 
-          "type": "object", 
+          "type": "object",
+          "description": "Type of the object",
+          "additionalProperties": false,
           "properties": {
             "Name": {
-              "$id": "/properties/Target/properties/Type/properties/Name", 
-              "type": "string", 
+              "type": "string",
               "enum": [
-                "literature", 
+                "literature",
                 "dataset",
-                "unknwon"
-              ], 
-              "title": "The Name Schema.", 
-              "description": "An explanation about the purpose of this instance.", 
-              "default": "", 
+                "unknown"
```

Comment:
Looks like 'unknown' is not a valid Type of a Source/Target, according to the documentation. Should this be removed?